### ISSUE TITLE
Fatal Exception java.lang.ArrayIndexOutOfBoundsException

### ISSUE THREAD

**Issue by daviidmorales** [Lyokone/flutterlocation#301]
**Tested on:**
 - Android, API Level 24, real device

**Devices:**
- Galaxy A3 (2017)
- Galaxy J5 Prime
- Galaxy A7(2017)
- Galaxy J7 Prime
- Galaxy J7(2016)

**Additional logs**

``` shell
Fatal Exception: java.lang.ArrayIndexOutOfBoundsException: length=7; index=9
       at com.lyokone.location.LocationPlugin$3.onNmeaMessage(LocationPlugin.java:303)
       at android.location.LocationManager$GnssStatusListenerTransport$GnssHandler.handleMessage(LocationManager.java:1419)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6776)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1518)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1408)
```

**Comment by Lyokone** [comment]
You do realize that your issue is totally empty of any meaningful information for me to help you ?

**Comment by Lyokone** [comment]
Which version of the plugin are you using ?

**Comment by daviidmorales** [comment]
^2.3.5

**Comment by Lyokone** [comment]
Was fixed in 2.5 release :)
